glide.yaml

```diff
@@ -1,265 +1,187 @@
-    version: 4eba002a5eaea69cf8d235a388fc6b65ae68d2dd
-
-  # START_PROMETHEUS_DEPS
-  - package: github.com/prometheus/prometheus
-    version: ~2.12.0
-
-  # To avoid prometheus/prometheus dependencies from breaking,
-  # pin the transitive dependencies
-  - package: github.com/prometheus/common
-    version: ~0.7.0
-  # END_PROMETHEUS_DEPS
-
-  # START_TALLY_PROMETHEUS_DEPS
-  - package: github.com/m3db/prometheus_client_golang
-    version: 8ae269d24972b8695572fa6b2e3718b5ea82d6b4
-
-  - package: github.com/m3db/prometheus_client_model
-    version: 8b2299a4bf7d7fc10835527021716d4b4a6e8700
-
-  - package: github.com/m3db/prometheus_common
-    version: 25aaa3dff79bb48116615ebe1dea6a494b74ce77
-
-  - package: github.com/m3db/prometheus_procfs
-    version: 1878d9fbb537119d24b21ca07effd591627cd160
-  # END_PROMETHEUS_DEPS
-
-  - package: github.com/coreos/pkg
-    version: 4
-    subpackages:
-      - capnslog
-
-  # START_TRACING_DEPS
-  - package: github.com/opentracing/opentracing-go
-    version: ^1.1.0
-
-  - package: github.com/uber/jaeger-lib
-    version: ^2.0.0
-
-  - package: github.com/uber/jaeger-client-go
-    version: ~2.16.0
-
-  - package: github.com/lightstep/lightstep-tracer-go
-    version: ~v0.18.0
-
-  - package: github.com/lightstep/lightstep-tracer-common
-    version: ~1.0.3
-
-  - package: github.com/opentracing-contrib/go-stdlib
-    # Pin this on recommendation of the repo (no stable release yet). Still arguably better than rewriting
-    # the same code.
-    version: cf7a6c988dc994e945d2715565026f3cc8718689
-
-  # END_TRACING_DEPS
-
-  # To avoid conflicting packages not resolving the latest GRPC
-  - package: google.golang.org/grpc
-    version: 1.7.5
-    subpackages:
-      - codes
-
-  - package: gopkg.in/validator.v2
-    version: 3e4f037f12a1221a0864cf0dd2e81c452ab22448
-    repo: https://github.com/go-validator/validator.git
-    vcs: git
-
-  - package: gopkg.in/go-playground/validator.v9
-    version: a021b2ec9a8a8bb970f3f15bc42617cb520e8a64
-    repo: https://github.com/go-playground/validator.git
-    vcs: git
-
-  - package: github.com/go-playground/universal-translator
-    version: 71201497bace774495daed26a3874fd339e0b538
-
-  - package: gopkg.in/yaml.v2
-    version: 5420a8b6744d3b0345ab293f6fcba19c978f1183
-    repo: https://github.com/go-yaml/yaml.git
-    vcs: git
-
-  - package: github.com/russross/blackfriday
-    version: ^2.0.1
-
-  - package: github.com/mauricelam/genny
-    version: eb2c5232c885956af3565a20ecf48555cab2b9bc
-
-  - package: github.com/leanovate/gopter
-    version: e2604588f4db2d2e5eb78ae75d615516f55873e3
-
-  - package: github.com/rakyll/statik
-    version: ^0.1.6
-
-  - package: golang.org/x/sys
-    subpackages:
-      - unix
-    version: c178f38b412c7b426e4e97be2e75d11ff7b8d4d4
-
-  - package: go.uber.org/config
-    version: ^1.3.1
-
-  - package: github.com/twotwotwo/sorts
-    version: bf5c1f2b8553dec28372aa6ac5a8bf53a20a4c5b
+- package: github.com/influxdata/influxdb
```

Comment:
Unfortunately we'll have to limit the modifications to this file since it can cause a significant change in performance if a lot of core dependencies change their profiles (which is hard to do with every single change).

What's the concrete dependencies we need to update/add with this change?  Let us know and we can scope to just updating those.